std/algorithm/iteration.d

```diff
@@ -2695,12 +2743,110 @@ version (none) // this example requires support for non-equivalence relations
     }
 }
 
-// https://issues.dlang.org/show_bug.cgi?id=13595
-version (none) // This requires support for non-equivalence relations
+
+
+// https://issues.dlang.org/show_bug.cgi?id=13805
+@system unittest
+{
+    [""].map!((s) => s).chunkBy!((x, y) => true);
+}
+
+/**
+Splits a forward range into subranges in places determined by a binary
+predicate.
+
+When iterating, one element of `r` is compared with `pred` to the next
+element. If `pred` return true, a new subrange is started for the next element.
+Otherwise, they are part of the same subrange.
+
+If the elements are compared with an unequivalency (!=) operator, it's recommended
+to consider $(LREF chunkBy) instead, as it's likely to take less time to execute.
+
+Params:
+pred = Predicate for determining where to split. The earlier element in the
+source range is always given as the first argument.
+r = A $(REF_ALTTEXT forward range, isForwardRange, std,range,primitives) to be split.
+Returns: a range of ranges in which
+all elements in a given subrange are `false` when compared to the next element
+with the predicate. The range is currently a reference range, but this may
+change in the future.
```

Comment:
This one's tough. Here's my best attempt:
```suggestion
Returns: a range of subranges of `r`, split such that within a given subrange,
calling `pred` with any pair of adjacent elements as arguments returns `false`.
Copies of the range currently have reference semantics, but this may change in
the future.
```